src/components/Container/index.tsx

```diff
@@ -36,7 +36,7 @@ const Container = () => {
 
 			<div className={s.content}>
 				<div className={s.header}>
-					<HeaderTest />
+					<Header />
```

Comment:
do u need a div wrapper above the header?  u should add styles to a div wrapper inside the header. it will be more clearly structured